test/testdata/cgo/main.go

```diff
@@ -14,6 +14,7 @@ import (
 	"unsafe"
 )
 
+//nolint:nosnakecase
```

Comment:
```suggestion
```

Reply:
If the directive is deleted, CI  is failed as the following reason.

```console
--- PASS: TestTestsAreLintedByDefault (2.52s)
=== RUN   TestCgoOk
level=info msg="[test] stderr: "
level=info msg="[test] ran [../golangci-lint run --go=1.17 --internal-cmd-test --allow-parallel-runners --no-config --enable-all testdata/cgo] in 3.078396167s"
    testshared.go:53: 
        	Error Trace:	testshared.go:53
        	            				run_test.go:98
        	Error:      	Not equal: 
        	            	expected: ""
Error:         	            	actual  : "testdata/cgo/main.go:18:8: _Cfunc_CString is used under score. You should use mixedCap or MixedCap. (nosnakecase)\n\tcs := C.CString(\"Hello from stdio\\n\")\n\t      ^\ntestdata/cgo/main.go:19:2: _Cfunc_myprint is used under score. You should use mixedCap or MixedCap. (nosnakecase)\n\tC.myprint(cs)\n\t^\ntestdata/cgo/main.go:20:11: _cgo0 is used under score. You should use mixedCap or MixedCap. (nosnakecase)\n\tC.free(unsafe.Pointer(cs))\n\t         ^\n"
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1,10 @@
Error:         	            	+testdata/cgo/main.go:18:8: _Cfunc_CString is used under score. You should use mixedCap or MixedCap. (nosnakecase)
        	            	+	cs := C.CString("Hello from stdio\n")
        	            	+	      ^
Error:         	            	+testdata/cgo/main.go:19:2: _Cfunc_myprint is used under score. You should use mixedCap or MixedCap. (nosnakecase)
        	            	+	C.myprint(cs)
        	            	+	^
Error:         	            	+testdata/cgo/main.go:20:11: _cgo0 is used under score. You should use mixedCap or MixedCap. (nosnakecase)
        	            	+	C.free(unsafe.Pointer(cs))
        	            	+	         ^
        	Test:       	TestCgoOk
        	Messages:   	exit code is 1
    testshared.go:54: 
        	Error Trace:	testshared.go:54
        	            				run_test.go:98
        	Error:      	Not equal: 
        	            	expected: 0
        	            	actual  : 1
        	Test:       	TestCgoOk
Error:         	Messages:   	output is testdata/cgo/main.go:18:8: _Cfunc_CString is used under score. You should use mixedCap or MixedCap. (nosnakecase)
        	            		cs := C.CString("Hello from stdio\n")
        	            		      ^
Error:         	            	testdata/cgo/main.go:19:2: _Cfunc_myprint is used under score. You should use mixedCap or MixedCap. (nosnakecase)
        	            		C.myprint(cs)
        	            		^
Error:         	            	testdata/cgo/main.go:20:11: _cgo0 is used under score. You should use mixedCap or MixedCap. (nosnakecase)
        	            		C.free(unsafe.Pointer(cs))
```